streams/src/test/java/org/apache/kafka/streams/state/internals/MeteredKeyValueStoreTest.java

```diff
@@ -91,13 +103,18 @@
     private KeyValueStore<Bytes, byte[]> inner;
     @Mock(type = MockType.NICE)
     private InternalProcessorContext context;
+    @Mock(type = MockType.NICE)
+    private Serde<String> keySerde;
+    @Mock(type = MockType.DEFAULT)
+    private Serializer<String> keySerializer;
+    @Mock(type = MockType.NICE)
+    private Serde<String> valueSerde;
+    @Mock(type = MockType.DEFAULT)
+    private Deserializer<String> valueDeserializer;
+    @Mock(type = MockType.DEFAULT)
+    private Serializer<String> valueSerializer;
```

Comment:
It looks like these are only used in one method. Can we define them inline instead?

This feedback also applies to the other tests.

Reply:
Ack